supabase/migrations/20240313073422_init.sql

```diff
@@ -0,0 +1,1027 @@
+    "amount" text not null,
+    "nonce" text not null,
+    "deadline" text not null,
+    "signature" character(132) not null,
+    "token_id" integer,
+    "partner_id" integer,
+    "beneficiary_id" integer not null,
+    "transaction" character(66),
+    "location_id" integer
+);
+
+
+alter table "public"."permits" enable row level security;
+
+create table "public"."settlements" (
+    "id" integer not null default nextval('settlements_id_seq'::regclass),
+    "created" timestamp with time zone not null default now(),
+    "updated" timestamp with time zone,
+    "user_id" integer not null,
+    "location_id" integer,
+    "credit_id" integer,
+    "debit_id" integer
+);
+
+
+alter table "public"."settlements" enable row level security;
+
+create table "public"."tokens" (
+    "id" integer not null default nextval('tokens_id_seq'::regclass),
+    "created" timestamp with time zone not null default now(),
+    "updated" timestamp with time zone,
+    "network" smallint not null default nextval('new_tokens_network_seq'::regclass),
+    "address" character(42) not null,
+    "location_id" integer
+);
+
+
+alter table "public"."tokens" enable row level security;
+
+create table "public"."users" (
+    "id" integer not null default nextval('users_id_seq'::regclass),
+    "created" timestamp with time zone not null default now(),
+    "updated" timestamp with time zone,
+    "wallet_id" integer,
+    "location_id" integer
+);
+
+
+alter table "public"."users" enable row level security;
+
+create table "public"."wallets" (
+    "id" integer not null default nextval('wallets_id_seq'::regclass),
+    "created" timestamp with time zone not null default now(),
+    "updated" timestamp with time zone,
+    "address" character(42),
+    "location_id" integer
+);
+
+
+alter table "public"."wallets" enable row level security;
+
+alter sequence "public"."credits_id_seq" owned by "public"."credits"."id";
+
+alter sequence "public"."location_id_seq1" owned by "public"."locations"."id";
+
+alter sequence "public"."new_access_id_seq" owned by "public"."access"."id";
+
+alter sequence "public"."new_debits_id_seq" owned by "public"."debits"."id";
+
+alter sequence "public"."new_logs_id_seq" owned by "public"."logs"."id";
+
+alter sequence "public"."new_partners_id_seq" owned by "public"."partners"."id";
+
+alter sequence "public"."new_permits_id_seq" owned by "public"."permits"."id";
+
+alter sequence "public"."new_tokens_id_seq" owned by "public"."tokens"."id";
+
+alter sequence "public"."new_tokens_network_seq" owned by "public"."tokens"."network";
+
+alter sequence "public"."new_users_id_seq" owned by "public"."users"."id";
+
+alter sequence "public"."new_wallets_id_seq" owned by "public"."wallets"."id";
+
+alter sequence "public"."settlements_id_seq" owned by "public"."settlements"."id";
+
+alter sequence "public"."unauthorized_label_changes_id_seq" owned by "public"."labels"."id";
+
+CREATE UNIQUE INDEX credits_pkey ON public.credits USING btree (id);
```

Comment:
it'd be much easier to read this if unique was defined in the table itself

Reply:
I agree. But since this is jsimply a dump from [supabase db dump](https://supabase.com/docs/reference/cli/supabase-db-dump) which is why everything is formatted this way. Since this db never had a single migration before it is hard for me to keep track of everything that was changed and I thought it would be less likely to miss anything that way